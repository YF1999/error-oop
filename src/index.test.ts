import * as Errors from './index';

test('Test all exports', () => {
    expect(Object.keys(Errors)).toMatchSnapshot();
});
