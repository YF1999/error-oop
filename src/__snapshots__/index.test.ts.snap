// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test all exports 1`] = `
Array [
  "NativeError",
  "Error",
  "ErrorTool",
  "NativeRangeError",
  "RangeError",
  "NativeReferenceError",
  "ReferenceError",
  "NativeSyntaxError",
  "SyntaxError",
  "NativeTypeError",
  "TypeError",
  "NativeURIError",
  "URIError",
  "AlreadyInUseError",
  "AuthenticationError",
  "AuthenticationRequiredError",
  "ConnectionError",
  "IndexOutOfRangeError",
  "InvalidOperationError",
  "NotFoundError",
  "NotImplementedError",
  "NotPermittedError",
  "NotSupportedError",
  "OperationCanceledError",
  "OutOfMemoryError",
  "StackOverflowError",
  "TimeoutError",
  "UnauthorizedAccessError",
  "UnauthorizedError",
  "ArgumentError",
  "ArgumentNullError",
  "ArgumentOutOfRangeError",
  "DataError",
  "TransactionError",
  "DirectoryNotFoundError",
  "DriveNotFoundError",
  "EndOfStreamError",
  "FileLoadError",
  "FileNotFoundError",
  "IOError",
  "PathTooLongError",
  "SocketError",
]
`;
